bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -553,6 +553,10 @@ private function addAssetNode(ArrayNodeDefinition $rootNode): void
                                 ->defaultTrue()
                                 ->info('Process text for Asset documents (e.g. used by backend search).')
                             ->end()
+                            ->booleanNode('scan_pdf')
```

Comment:
What about using a more generic name, like `vulnerability_scan` - maybe we need to implement this for other types as well 🤔

Reply:
yes, can be changed to more generic name. I'm not sure though what other types should be considered.

Reply:
Only if we plan to preview html directly in Admin UI and this will definitely have some conflicts  with CSP headers set for Admin.

Reply:
True - so for me it's also fine to keep the current naming 😊